providers/server_monitor.rb

```diff
@@ -35,7 +35,12 @@ def install_newrelic_service_linux
   package new_resource.service_name do
     action new_resource.action
   end
+
   # configure your New Relic license key
+  if node.platform_family.include?('debian')
+    execute "dpkg-divert --add --local --rename #{new_resource.config_path}/nrsysmond.cfg"
+  end
+
```

Comment:
Would you mind explaining a bit more in detail why this is needed for the browser_monitoring_capture_attributes-change?

If possible, revert this bit and then I'm happy to merge this PR in!